test/templates/api/repositories/template-inputs/initial-state.json

```diff
@@ -0,0 +1,5 @@
+{
+  "backend": {
+    "type": "snowflake"
+  }
+}
```

Comment:
There were some unstable E2E Templates API tests - they didn't work with GCP test project, because Snowflake Writer is missing (it is used in production templates, which are loaded on the API startup).